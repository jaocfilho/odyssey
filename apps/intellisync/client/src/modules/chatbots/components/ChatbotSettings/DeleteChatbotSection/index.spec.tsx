import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';

import { DeleteChatbotSection } from '.';

describe('DeleteChatbotSection', () => {
  it('should render without crashing', () => {
    const { container } = render(<DeleteChatbotSection />);
    expect(container).toMatchSnapshot();
  });
});
