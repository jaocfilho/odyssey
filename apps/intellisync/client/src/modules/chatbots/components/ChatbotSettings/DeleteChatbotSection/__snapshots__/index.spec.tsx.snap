// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DeleteChatbotSection > should render without crashing 1`] = `
<div>
  <div
    class="grid max-w-7xl grid-cols-1 gap-x-8 gap-y-10 px-4 py-16 sm:px-6 md:grid-cols-3 lg:px-8"
  >
    <div>
      <h2
        class="text-base font-semibold leading-7"
      >
        Delete chatbot
      </h2>
      <p
        class="mt-1 text-sm leading-6 dark:text-grayScheme-400"
      >
        This action is not reversible. All information related to this chatbot will be deleted permanently.
      </p>
    </div>
    <div
      class="flex items-start md:col-span-2"
    >
      <button
        class="flex font-semibold text-white shadow-sm justify-center focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 disabled:opacity-50 rounded-md px-2.5 py-1.5 text-sm dark:bg-red-500 dark:hover:bg-red-400 dark:focus-visible:outline-red-500 bg-red-600 hover:bg-red-500 focus-visible:outline-red-600 disabled:dark:bg-red-700"
        type="button"
      >
        Delete my chatbot
      </button>
    </div>
  </div>
</div>
`;
